framework/encode/capture_manager.cpp

```diff
@@ -497,7 +507,8 @@ void CaptureManager::CheckContinueCaptureForWriteMode()
         }
     }
     else if (IsTrimHotkeyPressed() ||
-             ((trim_key_frames_ > 0) && (current_frame_ >= (trim_key_first_frame_ + trim_key_frames_))))
+             ((trim_key_frames_ > 0) && (current_frame_ >= (trim_key_first_frame_ + trim_key_frames_))) ||
+             !IsTrimEnvVarEnabled())
```

Comment:
This breaks desktop range capture. Desktop hotkey captures now only last one frame long because `!IsTrimEnvVarEnabled()` always evaluates to true.

Reply:
Thanks for reporting this. Indeed this line causes a regression with the desktop hot key